README.md

```diff
@@ -112,15 +114,15 @@ const receiver = new FiveBellsLedgerPlugin({
 
 const receiverSecret = Buffer.from('secret_seed')
 const { sharedSecret, destinationAccount } = ILP.PSK.generateParams({
-  destinationAccount: receiver,
+  destinationAccount: 'us.usd.blue.bob',
```

Comment:
I think it's better to put `reciever.getAccount()` because otherwise it's unclear how you'd figure that address out